aztec/src/main/kotlin/org/wordpress/aztec/Aztec.kt

```diff
@@ -44,6 +51,10 @@ open class Aztec private constructor(val visualEditor: AztecText, val sourceEdit
                  toolbar: AztecToolbar, toolbarClickListener: IAztecToolbarClickListener) : Aztec {
             return Aztec(visualEditor, sourceEditor, toolbar, toolbarClickListener)
         }
+
+        fun with(visualEditor: AztecText) : Aztec {
```

Comment:
I think it would be useful to add an argument for the toolbar and toolbar listener (with default null value), so that you can use the visual editor with or without a toolbar.